awx/main/utils/external_logging.py

```diff
@@ -15,6 +15,7 @@ def construct_rsyslog_conf_template(settings=settings):
     host = getattr(settings, 'LOG_AGGREGATOR_HOST', '')
     port = getattr(settings, 'LOG_AGGREGATOR_PORT', '')
     protocol = getattr(settings, 'LOG_AGGREGATOR_PROTOCOL', '')
+    type = getattr(settings, 'LOG_AGGREGATOR_TYPE', '')
```

Comment:
Let's pick a different name that isn't a Python builtin.

Reply:
Good call, that is fixed now (there and in the tests).